modules/33acrossAnalyticsAdapter.js

```diff
@@ -334,21 +349,67 @@ function analyticEventHandler({ eventType, args }) {
     case EVENTS.AUCTION_INIT:
       onAuctionInit(args);
       break;
-    case EVENTS.BID_REQUESTED:
+    case EVENTS.BID_REQUESTED: // set bid.status: 'pending'
       onBidRequested(args);
       break;
+    case EVENTS.BID_TIMEOUT:
+      setBidStatus(getCachedBid(args.auctionId, args.bidId), 'timeout');
```

Comment:
my recommendation here is to create another function that does a couple things:
1- Get the cached bid
2. set the bid status.

the name of the fn could be something like `setCachedBidStatus`

also, I'll recommend to have an enum of all the different statuses.
e.g.
```
const BidStatus = {
  PENDING: 'pending'
}
...
case EVENTS.BID_TIMEOUT:
  setCachedBidStatus(args.auctionId, args.bidId, BidStatus.PENDING)
 ```

Reply:
Great idea. Done.